examples/meshgrid_voidanalysis_MFI.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+from carmm.meshgrid.meshgrid_unitcell import UnitCell
+from carmm.meshgrid.meshgrid_functions import atom_mesh_build_mask
+from carmm.meshgrid.meshgrid_void import void_find
+from carmm.meshgrid.meshgrid_void import void_build_mask
+from carmm.meshgrid.meshgrid_void import void_analysis
+import matplotlib.pyplot as plt
+from ase import io
+
+# An example showing how to plot the total volume not occupied by molecular vdW volume.
```

Comment:
For some reason with the QA testing (and don't ask why), test cases have to be setup in a function and then that function itself called. See e.g. https://github.com/logsdail/carmm/blob/master/examples/analyse_angles.py#L9 for a working example. I've no problem with the example here, but can it be changed into the format highlighted?